config/env/production/middlewares.js

```diff
@@ -0,0 +1,22 @@
+module.exports = [
```

Comment:
I might be reading this (https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.html#middlewares-configuration) wrong but it seems strapi only reads from `config/middlewares.js` and not `config/env`?

Reply:
This works in v4. I've set it this way to use cloudinary only on production. But i guess there this could be simplified by using cloudinary on dev environment too.

Reply:
![Sun Feb 20 06:38:37 PM IST 2022](https://user-images.githubusercontent.com/4880359/154844254-25d5370b-f5e9-465f-8962-55515e04bfc4.png)
![Sun Feb 20 06:39:17 PM IST 2022](https://user-images.githubusercontent.com/4880359/154844258-acf9e534-d48d-4412-a507-a5634b682179.png)